study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/viewSpatialStudyActiveTask.jsp

```diff
@@ -2324,14 +2332,14 @@
       if (charCode == 16)
         isShift = false;
       if (!isShift && $(this).val()) {
-        var regularExpression = /^[A-Za-z0-9*()_+|:.-]*$/;
+        var regularExpression = /^[a-z0-9_-]*$/;
         if (!regularExpression.test($(this).val())) {
-          var newVal = $(this).val().replace(/[^A-Za-z0-9\*\(\)_+|:.\-]/g, '');
+          var newVal = $(this).val().replace(/[^a-z0-9_-]/g, '');
           e.preventDefault();
           $(this).val(newVal);
           $(this).parent().addClass("has-danger has-error");
           $(this).parent().find(".help-block").empty().append($("<ul><li> </li></ul>").attr("class","list-unstyled").text(
-              "The characters like (< >) are not allowed."));
+        		  "Please use allowed characters only: lowercase alphabets (a-z), digits (0-9), _ (underscore) and -(minus)."));
```

Comment:
sorry, same change here: `lowercase letters` and space next to the `-`

Reply:
Fixed review comment.